frame/vesting/src/lib.rs

```diff
@@ -346,11 +354,14 @@ impl<T: Config> Pallet<T> {
 	}
 
 	/// Returns locked balance based on current block number.
+	/// TODO: add vesting_schedule_id parameter
```

Comment:
IMO it might be better to just return a total balance and balances grouped by vesting_schedule_id

Reply:
Yes, this makes sense. Updated.